playbook/app/pb_kits/playbook/pb_distribution_bar/_distribution_bar.jsx

```diff
@@ -18,13 +19,13 @@ const normalizeCharacters = (widths) => {
   })
 }
 
-const barValues = (normalizedValues) => {
+const barValues = (normalizedValues, colors) => {
   const arrSum = (value) => value.reduce((a, b) => a + b, 0)
   const widthSum = arrSum(normalizedValues)
   return normalizedValues.map((value, i) => {
     return (
       <div
-          className="pb_distribution_width"
+          className={classnames('pb_distribution_width', colors[i] ? `color_${colors[i]}` : '')}
```

Comment:
```suggestion
          className={classnames('pb_distribution_width', colors.hasOwnProperty(i) ? `color_${colors[i]}` : '')}
```

I think this might fail if you directly try to access a key by name. Instead you can check for it this way.

Reply:
Looks like `colors` is an array not an object.